.github/workflows/check-using-docker.yaml

```
@@ -1,55 +1,62 @@
 
-        steps:
-          # Note : We can not use reusable workflow when the job runs on a container. Only Github Actions can be used.
-          - name: Checkout repo
-            uses: actions/checkout@v4.1.1
-            with:
-              path: ${{ github.event.repository.name }}
+      steps:
+        # Note : We can not use reusable workflow when the job runs on a container. Only Github Actions can be used.
+        - name: Checkout repo
+          uses: actions/checkout@v4.1.1
+          with:
+            path: ${{ github.event.repository.name }}
 
-          - name: Get package name 📦
-            id: package-name
-            run: |
-              PKGBUILD="$(Rscript -e 'cat(sprintf("%s_%s.tar.gz",(dcf <- read.dcf("DESCRIPTION"))[,"Package"], dcf[,"Version"]))')"
-              echo "PKGBUILD = $PKGBUILD"
-              echo "pkgbuild=$PKGBUILD" >> $GITHUB_OUTPUT
-            shell: bash
-            working-directory: ${{ github.event.repository.name }}
+        - name: Get package name 📦
+          id: package-name
+          run: |
+            PKGBUILD="$(Rscript -e 'cat(sprintf("%s_%s.tar.gz",(dcf <- read.dcf("DESCRIPTION"))[,"Package"], dcf[,"Version"]))')"
+            echo "PKGBUILD = $PKGBUILD"
+            echo "pkgbuild=$PKGBUILD" >> $GITHUB_OUTPUT
+          shell: bash
+          working-directory: ${{ github.event.repository.name }}
 
-          # Build a package stored in a directory called ${{ github.event.repository.name }}
-          # --no-manual --no-build-vignettes because this requires LaTeX.
-          - name: Build R package 🏗
-            run: |
-              R CMD build --no-manual --no-build-vignettes ${{ github.event.repository.name }}
-            shell: bash
+        # Build a package stored in a directory called ${{ github.event.repository.name }}
+        # --no-manual --no-build-vignettes because this requires LaTeX.
+        - name: Build R package 🏗
+          run: |
+            R CMD build --no-manual --no-build-vignettes ${{ github.event.repository.name }}
+          shell: bash
 
-          # Check `.tar.gz` package built in previous step.
-          # --no-manual --no-vignettes because this requires LaTeX.
-          - name: Check R package 🏁
-            run: |
-              echo "Let's run R CMD check..."
-              R CMD check --no-manual --no-build-vignettes ${{ steps.package-name.outputs.pkgbuild }}
-            shell: bash
\ No newline at end of file
+        # Check `.tar.gz` package built in previous step.
+        # --no-manual --no-vignettes because this requires LaTeX.
+        - name: Check R package 🏁
+          run: |
+            echo "Let's run R CMD check..."
+            R CMD check --no-manual --no-build-vignettes ${{ steps.package-name.outputs.pkgbuild }}
+          shell: bash
+
+        # Print dependencies
+        - name: Print deps
+          run: |
+            echo "Let's print installed dependencies :"
+            installed.packages()
+          shell: bash
```

I think this should run in `Rscript` shell, right?
```suggestion
          run: |
            cat("Let's print installed dependencies:\n")
            installed.packages()
          shell: Rscript {0}
```

---

Yes definitely .. thanks for fixing this :)